docker/install/ubuntu_install_papi.sh

```diff
@@ -23,11 +23,12 @@ set -o pipefail
 apt-get update --fix-missing
 
 # deps
-apt-get install -y linux-tools-common linux-tools-generic
+apt-get install -y linux-tools-common linux-tools-generic kmod
 
 cd /
 git clone https://bitbucket.org/icl/papi.git
 cd papi/src
```

Comment:
could we also pin to the current hash here with `git checkout <hash>` or something so this doesn't randomly pop up in the future?

Reply:
Yes, we can. I wasn't sure if we wanted to checkout a fixed commit, and can't see any drawbacks to doing this.

Reply:
we haven't been super great about it but I think we should be doing it everywhere, if we want to be rebuilding the docker images from the base PR every time to avoid the 2-PRs-to-update-Docker problem we need to get them as stable as we can